pkg/logs/config/endpoints.go

```diff
@@ -93,3 +94,26 @@ func NewEndpointsWithBatchSettings(main Endpoint, additionals []Endpoint, usePro
 		BatchMaxContentSize:    batchMaxContentSize,
 	}
 }
+
+// GetReliableAdditionals returns additional endpoints that can be failed over to and block the pipeline in the
+// event of an outage and will retry errors. These endpoints are treated the same as the main endpoint.
+func (e *Endpoints) GetReliableAdditionals() []*Endpoint {
+	endpoints := []*Endpoint{}
+	for _, endpoint := range e.Additionals {
+		if endpoint.IsReliable {
+			endpoints = append(endpoints, &endpoint)
+		}
+	}
+	return endpoints
```

Comment:
Because we currently only support one reliable endpoint, in my opinion we should log something here, or later in the pipeline to explicitly mention only the first reliable endpoint in the list will be honored.